lib/cdo/script_constants.rb

```diff
@@ -18,26 +18,31 @@ module ScriptConstants
   INFINITY_NAME = 'infinity'
   ARTIST_NAME = 'artist'
   ALGEBRA_NAME = 'algebra'
+  ALGEBRA_A_NAME = 'AlgebraA'
+  ALGEBRA_B_NAME = 'AlgebraB'
   FLAPPY_NAME = 'flappy'
   TWENTY_HOUR_NAME = '20-hour'
   ACCELERATED_NAME = 'accelerated'
   COURSE1_NAME = 'course1'
   COURSE2_NAME = 'course2'
   COURSE3_NAME = 'course3'
   COURSE4_NAME = 'course4'
+  CSP_UNIT1_NAME = 'cspunit1'
+  CSP_UNIT2_NAME = 'cspunit2'
+  CSP_UNIT3_NAME = 'cspunit3'
+  CSP_UNIT4_NAME = 'cspunit4'
+  CSP_UNIT5_NAME = 'cspunit5'
+  CSP_UNIT6_NAME = 'cspunit6'
```

Comment:
While they do match the values in Script.name, they're used more as IDs than "names". Refactoring the model as well would be a pretty large undertaking though, so I think it's better to just leave it as is.